config/initializers/content_security_policy.rb

```diff
@@ -16,6 +16,8 @@
     attachments_host = nil
   end
 
+  hcaptcha_hosts = ["https://hcaptcha.com", "https://*.hcaptcha.com"]
```

Comment:
Maybe only add it if the feature is enabled.

Reply:
resolved in next commit (feel free to reopen if you’re not happy with the ternary assignement? idk if it fits in the style)